web-console/4.0-designs/storage/snapshots/snapshot.md

```diff
@@ -0,0 +1,105 @@
+# PVC/PV Snapshots
+ 
+## Main use cases
+Create snapshot (from where PVC is)
+List / View related snapshot(s) for a given PVC
+Delete snapshot(s)
+Create new PVC from a snapshot (import/restore snapshot)
+ 
+## User Stories
```

Comment:
@yuvalgalanti Might be worth putting the user stories as bullets as "View File" does not render properly.